corehq/apps/auditcare/migrations/0001_sqlmodels.py

```diff
@@ -0,0 +1,113 @@
+# Generated by Django 2.2.16 on 2021-03-17 18:03
+
+import corehq.apps.auditcare.models
+import corehq.util.models
+from architect.commands import partition
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def add_partitions(apps, schema_editor):
+    partition.run({'module': 'corehq.apps.auditcare.models'})
+
+
+class Migration(migrations.Migration):
+
+    initial = True
+
+    dependencies = [
+    ]
+
+    operations = [
+        migrations.CreateModel(
+            name='HttpAccept',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('value', models.CharField(db_index=True, max_length=255)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='UserAgent',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('value', models.CharField(db_index=True, max_length=255)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='ViewName',
+            fields=[
+                ('id', models.AutoField(auto_created=True, primary_key=True, serialize=False, verbose_name='ID')),
+                ('value', models.CharField(db_index=True, max_length=255)),
+            ],
+        ),
+        migrations.CreateModel(
+            name='NavigationEventAudit',
+            fields=[
+                ('id', models.BigAutoField(primary_key=True, serialize=False)),
+                ('user', models.CharField(blank=True, max_length=255, null=True)),
+                ('domain', models.CharField(blank=True, max_length=126, null=True)),
+                ('event_date', models.DateTimeField(db_index=True, default=corehq.apps.auditcare.models.getdate)),
+                ('path', models.CharField(blank=True, default='', max_length=255)),
+                ('ip_address', models.CharField(blank=True, default='', max_length=45)),
+                ('session_key', models.CharField(blank=True, max_length=255, null=True)),
```

Comment:
I think it allows us to connect a series of requests in the same session. It is a historical artifact of auditcare that we are collecting it, and removing it felt out of scope for this refactor, although I could be persuaded otherwise.